dio/example/server/server.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:io';
+
+import 'package:shelf/shelf.dart';
+import 'package:shelf/shelf_io.dart';
+import 'package:shelf_router/shelf_router.dart';
+import 'package:shelf_cors_headers/shelf_cors_headers.dart';
+
+Future<void> main() async {
+  final handler = const Pipeline()
+      .addMiddleware(corsHeaders())
+      .addHandler(router);
+
+  final ip = InternetAddress.anyIPv4;
+  final port = int.parse(Platform.environment['PORT'] ?? '2384');
+  final server = await serve(
+    handler,
+    ip,
+    port,
+  );
+
+  print('Server listening on http://${server.address.host}:${server.port}');
```

Comment:
I'm curious (because I haven't experienced with `shelf`) whether `serve` will be ended when it's up or served ended.

Reply:
It should be terminated manually.

Reply:
If so, how do we integrate this into our continuous tests?

Reply:
So is the test run automatically?  If so, I think I should use your solution. (Remove the underscore and check if the value is set properly.)

Reply:
> So is the test run automatically? If so, I think I should use your solution. (Remove the underscore and check if the value is set properly.)

It surely does. See the failing checks below.

Reply:
Ok, I'll fix it with your solution.